aviary/subsystems/aerodynamics/gasp_based/test/test_gasp_aero_coeffs.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+import openmdao.api as om
+from openmdao.utils.assert_utils import assert_check_partials, assert_near_equal
+
+from aviary.subsystems.aerodynamics.gasp_based.gasp_aero_coeffs import AeroFormfactors
+from aviary.variable_info.variables import Aircraft, Mission
+
+tol = 1e-8
+
+
+class TestAeroCoeffs(unittest.TestCase):
+    def testAeroCoeffs(self):
```

Comment:
functions and methods should be snake_case, although for tests I don't care as much